tests/data/base/Speaker.php

```diff
@@ -40,6 +40,7 @@ public function scenarios()
     {
         return [
             'test' => ['firstName', 'lastName', '!underscore_style'],
+            'duplicates' => ['firstName', 'firstName', '!underscore_style', '!underscore_style'],
```

Comment:
no, sorry I had not seen the issue before commenting here so I was wondering why you put duplicate names in the scenario.